sips/sip-019/sip-019-modify-coinbase-reward-per-block-and-halving-schedule.md

```diff
@@ -0,0 +1,74 @@
+# Preamble
+
+SIP Number: 019
+
+Title: Modify Coinbase Reward per Block & Halving Schedule
+
+Author: Xan Ditkoff (email: xan@daemontechnologies.co)
+
+Consideration: Technical, Governance, Economics
+
+Type: Standard
+
+Status: Draft
+
+Created: 16 May 2022
+
+License: CC0-1.0
+
+Sign-off: 
+
+# Abstract
+There are currently two ways the Stacks network rewards STX miners when they win a block: STX tokens as part of a “Coinbase reward” and STX tokens from transaction fees. The purpose of the Coinbase reward is to bootstrap and to provide an extra incentive for miners to secure the network in the early days, when transaction fees are not yet sufficient. 
+
+For the first 10,000 blocks of the Stacks mainnet launch the coinbase reward was 2000 STX compared with 1000 STX today. This period coincided with significant interest from users and miners. Historically, the year 2050 STX supply has been reduced in the past (from 5B to 2B to 1.8B). This proposal calls for a slight increase in the year 2050 supply, and increasing the current coinbase reward to 1600 STX until the next halving. 
```

Comment:
The text here is confusing.  You're not just proposing increasing the current coinbase to 1600 STX until the next halvening; you're proposing increasing the scheduled coinbase emissions for _the next 14 years_ until landing at 100 STX/block.